devicetypes/smartthings/testing/simulated-window-shade.src/simulated-window-shade.groovy

```diff
@@ -31,6 +33,18 @@ metadata {
 		command "opened"
 		command "closed"
 		command "unknown"
+
+		command "shadeLevel0"
```

Comment:
What's the value in defining these custom commands?

Reply:
They were just shortcuts that one can use from Classic. Not sure that I have added some as tiles yet. I need to look back at this DTH.